internal/slack/conversation.go

```diff
@@ -42,7 +45,7 @@ func (s *client) findConversationByTitle(title string) ([]Conversation, error) {
 			Cursor: cursor,
 		}
 
-		channels, nextCursor, err := s.superBotClient.GetConversations(&params)
+		channels, nextCursor, err := s.botClient.GetConversations(&params)
```

Comment:
This method (`GetConversations`) is used when we want to refresh a specific submodule, and when we get a module payload through the message queue, we only get a slice of links. Actually, we use this method not because we need info about a conversation - we need conversation's ID to get its users later (we use it [here](https://github.com/acs-dl/slack-module-svc/blob/fix%2Flogs-and-billing/internal/slack/users_from_conversation.go)). So what we need is just conversation ID

One solution that immediately came to my mind is to save conversations along with their IDs in db when creating permissions (so this will happen each time the server starts or if somebody requests to refresh the whole module).
This way, instead of calling `GetConversations` and fetching all possible conversations, we can make a call to db. But if we don't find the convo we are looking for, we might still need to make a call to Slack API, because there is a chance somebody creates a new conversation in a workspace before the whole module refreshes.

What do you think about such a solution?

Reply:
Discussed in Telegram